License2Deploy/rolling_deploy.py

```diff
@@ -98,8 +98,9 @@ def get_autoscale_group_name(self):
 
   def get_autoscaling_group_name_from_cloudformation(self):
     if not self.autoscaling_groups:
-      self.autoscaling_groups = self.get_resources_from_stack_of_type('AWS::AutoScaling::AutoScalingGroup')
-    return self.get_resources_physical_ids_by_project(self.autoscaling_groups)[0]
+      asg_logical_name = '{0}ASG{1}'.format(self.project, self.env)
+      self.autoscaling_groups = self.cloudformation_client.describe_stack_resource(StackName=self.stack_name, LogicalResourceId=asg_logical_name)['StackResourceDetail']
+    return self.autoscaling_groups['PhysicalResourceId']
```

Comment:
Will this always be a single dictionary? if so can we remove the plurality in the var name for more clarity?